src/sdk/services/blockchain/contracts/VestingContract.ts

```diff
@@ -231,4 +324,80 @@ export class VestingContract {
 
         return totalBalance;
     }
+
+    getVestingCategory(categoryId: number): {
+        startDelay: number;
+        cliffPeriod: number;
+        vestingPeriod: number;
+        tgeUnlock: number;
+    } {
+        const vestingCategory = vestingCategories.get(categoryId);
+
+        if (!vestingCategory) {
+            throw new Error(`Vesting category ${categoryId} not found`);
+        }
+
+        return vestingCategory;
+    }
+
+    async getUnlockableBalance(
+        account: NameType
+    ): Promise<{ unlockable: number; locked: number; totalVested: number }> {
```

Comment:
also should return the vesting allocations and all the metadata needed for them for the UI.